platform-sdk/swirlds-unit-tests/common/swirlds-test-framework/src/main/java/com/swirlds/test/framework/config/TestConfigBuilder.java

```diff
@@ -154,14 +93,16 @@ public TestConfigBuilder withValue(@NonNull final String propertyName, final boo
      */
     @NonNull
     public TestConfigBuilder withValue(@NonNull final String propertyName, @NonNull final Object value) {
```

Comment:
I missed that we have the method with Object here, too. Again I would totally avoid it by now (even if I might have created this one).

Reply:
I reverted changes to this file in this PR. Perhaps we can update it in a future PR.